modules/aws/irsa/variables.tf

```diff
@@ -3,9 +3,12 @@ variable "name" {
   type        = string
 }
 
-variable "oidc_urls" {
-  description = "EKS OIDC URLs to trust"
-  type        = list(string)
+variable "oidc_providers" {
```

Comment:
I wrote this above, but could it be logical to make it a list of maps? To easier see which cluster it belongs to (as an example).